package.json

```diff
@@ -125,6 +125,7 @@
     "uuid": "^8.1.0",
     "walk-sync": "^2.1.0",
     "watch-detector": "^1.0.0",
+    "workerpool":"^3.1.1",
```

Comment:
The current release is 6.0.0, why use such an old version?

Reply:
Because that's what there in yarn.lock and I don't want to introduce any breaking changes.

Reply:
Hmm, but AFAICT we didn't use it before. If it was in the `yarn.lock`, it was from some transitive dependency right?

I _believe_ that this PR is ember-cli's first direct usage of `workerpool` (though we have used it indirectly for a while through `broccoli-babel-transpiler`), we should add the current version...